src/python_testing/TC_WEBRTCP_2_1.py

```diff
@@ -70,6 +73,11 @@ async def test_TC_WebRTCProvider_2_1(self):
         Executes the test steps for the WebRTC Provider cluster scenario.
         """
 
+        self.step("precondition")
+        # Commission DUT - already done
+        await self.allocate_one_audio_stream()
```

Comment:
I was wondering on which following steps we validated these allocated streams?

Reply:
The streams are designed to be valid, as we're not testing stream allocation, but the transport on top.  AVMS_* tests the actual stream allocation variances.

Reply:
If I comment out those two lines in the test scripts, will it impact the final test result?

Reply:
Yep, it'll fail as there's no allocated streams.  I can add checks to make sure the streams were actually allocated by the DUT.  Would that cover your comment?

Reply:
@yufengwangca I've added checks to make sure allocations were successful; as that's needed for the rest of the TCs.  This applies to all touched WEBRTCP TCs.  Can you take another look?